ts/DataGrid/Querying/QueryingController.ts

```diff
@@ -98,6 +98,16 @@ class QueryingController {
         this.sorting.loadOptions();
     }
 
+    /**
+     * Check if the data table does not need to be modified.
+     */
+    public willNotModify(): boolean {
+        return (
+            !this.sorting.modifier
+            // && !this.filtering.modifier
```

Comment:
To remove

Reply:
I left it in the querying templates for filtering to make it easier to add it later once `FilterModifier` is implemented.